helm/fiftyone-teams-app/values.yaml

```
@@ -433,6 +434,103 @@ delegatedOperatorExecutorSettings:
   # -- Volumes for delegated-operator-executor. [Reference][volumes].
   volumes: []
 
+# FiftyOne Teams Delegated Operator Executor (teams-do) configurations.
+delegatedOperatorInstances:
```

```suggestion
delegatedOperatorDeployments:
```
?  just a thought...